src/tests/source/wasm-particle-new.cc

```diff
@@ -0,0 +1,14 @@
+#include <arcs.h>
+
+class HotReloadTest : public arcs::Particle {
+  std::string getTemplate(const std::string& slot_name) override {
+    return R"(<div>Hello {{name}}, new age: {{age}}<\div>)";
```

Comment:
It doesn't affect this test, but for future reference: the '{{name}}' placeholders have to be contained in a HTML element, as you already have in the JS particle: `Hello <span>{{name}}</span>, old age: <span>{{age}}</span>`

Reply:
Changed! :)